packages/api-client/src/views/Request/RequestSidebar.vue

```diff
@@ -181,6 +189,21 @@ const handleClearDrafts = () => {
     }
   }
 }
+
+const isSearchVisible = ref(false)
+
+const toggleSearch = () => {
+  isSearchVisible.value = !isSearchVisible.value
+  if (isSearchVisible.value) {
+    // Use nextTick to ensure the input is mounted before focusing
+    nextTick(() => {
+      searchInputRef.value?.focus()
+    })
+  } else {
+    // Clear search text when toggling off
+    searchText.value = ''
```

Comment:
```suggestion
    searchText.value = ''
    isSearchVisible.value = false
```

could be nice to add this so it closes the search on click (right now it remains open)

Reply:
done